src/services/Safeplace.ts

```diff
@@ -0,0 +1,30 @@
+import IUser from '../components/interfaces/IUser';
+import axios from 'axios';
+
+const createHttpConfig = (token?: string) => axios.create({
+    baseURL: "http://api.safely-app.fr:8081",
+    headers: {
+        "Content-type": "application/json",
+        Authorization: `Bearer ${token}`
+    }
+});
+
+class Safeplace {
+    getAll(token: string) {
+        return createHttpConfig(token).get("/");
+    }
+
+    get(id: string, token: string) {
+        return createHttpConfig(token).get(`/safeplace/${id}`);
+    }
+
+    update(id: string, data: IUser, token: string) {
```

Comment:
Can we update Safeplaces from the administrator panel ?

Reply:
For the moment we cannot. We need to think about the requirements of this feature.

Maybe be able to:

- Desactivate / activate a safeplace.
- Delete a safe place.
- Manually add or edit scheduling...